db.js

```diff
@@ -1,5 +1,5 @@
 const Sequelize = require('sequelize')
 
-const sequelize = new Sequelize(`postgres://postgres:${process.env.DB_SECRET}@vpostgres.csmzuruqvoj3.us-west-2.rds.amazonaws.com:5432/Rad_Records`)
```

Comment:
Looks like we accidentally removed the environment variable here

Reply:
Ah... Not sure how that happened. I will make another commit later.